lib/plugins/slack/attachements.js

```diff
@@ -93,10 +93,20 @@ module.exports = {
       let color = 'danger';
 
       const limitMetric = metrics[options.limitMetric];
-      if (limitMetric) {
+      if (limitMetric && options.limitMetric === 'coach') {
         if (limitMetric.metric > options.limitError) {
           color = 'good';
-        } else if (limitMetric.metric > options.limitWarning) {
+        }
+        else if (limitMetric.metric > options.limitWarning) {
+          color = 'warning';
+        }
+      }
+      // SpeeddIndex/firstVisualChange
+      else if (limitMetric) {
+        if (limitMetric.metric < options.limitError) {
```

Comment:
here warning would be lower than error (we might want to verify that). Hence:
>= error: 'danger'
else if >= warning: 'warning'
(default to 'good')